src/Types/PgsqlIntegerRanges.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace staabm\PHPStanDba\Types;
+
+use PHPStan\Type\IntegerRangeType;
+use PHPStan\Type\IntegerType;
+use PHPStan\Type\Type;
+
+/**
+ * @see https://www.postgresql.org/docs/9.1/datatype-numeric.html
+ */
+final class PgsqlIntegerRanges
+{
+    public function smallint(): Type
```

Comment:
Does it also support signed/unsigned types? If so, more could be added after we landed this PR

Reply:
postgres doesn't support signed/unsigned integer, unless you "simulate" it with `CHECK (>=0)`. There's only serial/bigserial which "acts" like unsigned int, but actually is dependant of a sequence which defines the initial value and increment.